### ISSUE TITLE
Error getting sound

### ISSUE THREAD

**Issue by electrithm** [hiddentao/google-tts#18]
I am trying to add text to speech to my website but keep getting this error
```
GET http://translate.google.com/translate_tts?ie=UTF-8&tl=en&q=Ask%20Again&textlen=9&idx=0&total=1 404 (Not Found)
```

**Comment by electrithm** [comment]
Also, it works when I am running the file locally but doesn't work when its on the server

**Comment by hiddentao** [comment]
Yeah, it will only work locally or in a phonegap app, as it states in
the README.

**Comment by jabrena** [comment]
Today, I tried to test the project but I receive this http message:

When JS generate this address:
http://translate.google.com/translate_tts?ie=UTF-8&tl=es&q=demo&textlen=4&idx=0&total=1

google redirect to the following url:

``` bash
http://ipv4.google.com/sorry/IndexRedirect?continue=http://translate.google…%3D0%26total%3D1&q=CGMSBD4O5MQYgJ-gswUiGQDxp4NLmEQHEfv33en1moN-NPKk0aybPWc 503 (Service Unavailable)
```

Is a particular problem with me?

Juan Antonio

**Comment by blaiseliu** [comment]
I got the same error, @jabrena. Do we have a fix now?

**Comment by hiddentao** [comment]
Sorry guys, I haven't used this library in a while so I'm not sure. I wouldn't expect it to work unless you're calling from localhost or within phonegap/electron.js

**Comment by khanh2003** [comment]
*not work* 
|------|------|--------|

**Comment by roger2hk** [comment]
A token `tk` is required in the current API version.
`https://translate.google.com.hk/translate_tts?ie=UTF-8&q=%E6%B8%AC%E8%A9%A6&tl=zh-yue&total=1&idx=0&textlen=2&tk=395724.20302&client=t&ttsspeed=0.24`
